tests/simple.sh

```diff
@@ -101,6 +101,8 @@ add_failure () {
 # Test a no frills run with verbosity and make sure the expected items exist
 test_normal_report () {
     cmd="-vvv"
+    # get a list of initial kmods loaded
+    kmods=( $(lsmod | cut -f1 -d ' ') )
```

Comment:
We do rely on the same ordering of modules in the `lsmod` output; isnt it safer to `sort` them (here and in `new_komds` assignment)?

Reply:
afaik, the `sort` in the array comparison should be fine, but it would only help things to sort them on collection too, adding.